sdk/resourcemanager/Azure.ResourceManager.Core/README.md

```diff
@@ -0,0 +1,77 @@
+# Azure ResourceManager Core client library for .NET
```

Comment:
Can you open an ADO to come back revisit this readme. The content was originally aimed at RP packages not core.